typings/calendly.d.ts

```diff
@@ -0,0 +1,18 @@
+import { Props as BadgeWidgetOptions } from "./components/PopupWidget/PopupWidget";
```

Comment:
Whoops, looks like I forgot to add `typings` to `.gitignore` 😅 . Can you delete the `typings` directory and add `typings` to `.gitignore`?

Reply:
deleted and added!